lte/protos/mconfig/mconfigs.proto

```diff
@@ -71,6 +71,14 @@ message EnodebD {
         int32 tac = 8;
         int32 cell_id = 9;
         string ip_address = 10;
+        PowerControl power_control = 11;
+        bool x2_enable_disable = 12;
+        message PowerControl {
+          int32 reference_signal_power = 1;
+          uint32 power_class = 2;
+          uint32 pb = 3;
+          int32 pa = 4;
+        }
```

Comment:
Include some more detailed descriptions here, since this `.proto` file is a source of truth for all code relying on these definitions.